spec/models/storage_location_spec.rb

```diff
@@ -22,6 +22,26 @@
     end
   end
 
+  context "Callbacks >" do
+    describe "before_destroy" do
+      let!(:item) { create(:item) }
```

Comment:
```suggestion
      let(:item) { create(:item) }
```
You don't need to call `let!` since when you call `subject` it will call `item` and memoize it. This is a minor detail though and this still works